source/extensions/filters/http/ext_authz/ext_authz.h

```diff
@@ -268,6 +268,10 @@ class Filter : public Logger::Loggable<Logger::Id::filter>,
   struct PerRouteFlags {
     const bool skip_check_;
     const bool skip_request_body_buffering_;
+
+    static constexpr PerRouteFlags defaultFlags() { return PerRouteFlags{false, false}; }
```

Comment:
can we add comments on what the bools are here and below?

e.g., `false /*skip_check_*/, false /*skip_request_body_buffering_*/`